src/Integrations/Integrations/CodeIgniter/V2/CodeIgniterIntegration.php

```diff
@@ -70,7 +70,11 @@ function (SpanData $span) use ($rootSpan, $method, $service) {
                 $span->service = $service;
                 $span->type = Type::WEB_SERVLET;
 
-                $this->load->helper('url');
+                // We took the assumption that all controllers will extend CI_Controller.
+                // But we've at least seen one healthcheck controller not extending it.
+                if ($this->load) {
```

Comment:
Is this trying to guard against the presence of the `load` method? If so, it should be:

```suggestion
                if (\method_exists($this, 'load')) {
```

Reply:
I'll retry this, it was my first try and didn't work. But just in case I'll retry

Reply:
Actually, load don't seem to be a method right? it's initialized with `load_class`. I've added a check on the helper method while I was at it though inc75e1a80ea427923d66779fffa11e16bdf825e6b

Reply:
Doh! Your commit makes sense.